Makefile

```diff
@@ -70,6 +70,7 @@ testacc-dev-user:
 testacc-oss:
 	docker compose up -d
 	./scripts/testacc-oss $(TESTS)
+	docker compose down
```

Comment:
`-v`?

```
  -v, --volumes          Remove named volumes declared in the "volumes" section of the Compose file and anonymous volumes attached to containers
```

Reply:
I don't think we'll need that since there aren't any volumes created in this Compose setup - when I tested that locally, it still only deleted the network and the container